latex/Inhalt/grundlagen.tex

```diff
@@ -1,7 +1,5 @@
 \chapter{Grundlagen}\label{chap:prereqs}
 
-\section{Entwicklungsumgebung}
-
 \section{WebAssembly}
 
 "\ac{WASM} is a safe, portable, low-level code format designed for efficient execution and compact representation"\cite{WebAssemblyCoreSpecification}. Im Endeffekt handelt es sich bei \ac{WASM} also um eine low-level Bytecode-Sprache, die von Browsern ausgeführt werden kann. Diese Sprache soll ähnlich performant sein, wie die Ausführung naives Maschinen-Codes. Das Paper, in dem \ac{WASM} eingeführt wird, berichtet eine 10\% Performance-Diskrepanz zwischen \ac{WASM} und naivem Assembly\cite{10.1145/3062341.3062363}. Durch Kompilation nach \ac{WASM} ist es möglich Programme auf Seite des Clienten laufen zu lassen, die sonst vom Server ausgeführt werden müssten.
```

Comment:
Warum entfernst du das Zitat aber nicht die Quelle?

Reply:
Actually I don't. This is some weird highlighting done by Github (as you can see the lines on the left are not marked"

Reply:
lmao